ros2topic/ros2topic/api/__init__.py

```diff
@@ -164,38 +166,39 @@ def qos_profile_from_short_keys(
 ) -> rclpy.qos.QoSProfile:
     """Construct a QoSProfile given the name of a preset, and optional overrides."""
     # Build a QoS profile based on user-supplied arguments
+    if not preset_profile:
+        preset_profile = default_preset_str
```

Comment:
Since the default value seems to depend on the verb, I think it would be better to handle this case in `echo.py` where we can pass a default of "sensor_data" if `args.qos_profile` is `None`.

Reply:
Addressed on [160fce7](https://github.com/ros2/ros2cli/pull/613/commits/160fce7d4411ce53251d12ce92dd55b8e77fc046)